package.json

```diff
@@ -52,6 +52,7 @@
     "css-select-umd": "1.3.0-rc0",
     "diff": "3.5.0",
     "globalize": "1.4.0",
+    "immutable": "3.8.2",
```

Comment:
Should we include immutable as a dependency, or make it a dev dependency and also an optional dependency?

Reply:
I made it an optional and dev dependency. In the code I just wrapped the require in a `try`. I'm assuming that we expect that module will fail if Immutable isn't installed, but I'm not sure how we want to handle that. I didn't see anywhere else we're using optional dependencies so let me know if that's the wrong approach.

Reply:
Perhaps we should put something in the README to say that they would need to install immutable.... Or maybe having it as a dev isn't the end of the world... it's not going to affect anything unless they use it anyway.

Reply:
Your call. Since it should be left out if they're not using it I'd be fine with it being a dependency.

Reply:
yeah let's put it in the deps again. sorry!

Reply:
No problem. Reverted